cmd/migrate.go

```diff
@@ -24,6 +24,8 @@ import (
 	"path/filepath"
 	"strings"
 
+	"github.com/nats-io/nkeys"
+
```

Comment:
don't need new line here, which should then have it alpha ordered with the rest of the "github" imports.

Reply:
fixed